versions.tf

```
@@ -1,5 +1,5 @@
 terraform {
-  required_version = ">= 0.12"
+  required_version = ">= 0.13"
```

We aren't actually using any 0.13 features here, so this can stay 0.12.

```suggestion
  required_version = ">= 0.12"
```

---

Okay.  Will undo the change.